doc/createAvailableCommandlineCommandsListView.awk

```diff
@@ -0,0 +1,16 @@
+BEGIN {
+	# Switch field separator from space to tabulator.
+	FS="\t"	
+}
+
+{
+	print "'''"
+	print "[[cmd-" $1 "]]"
+	print "****"
+	print "*Syntax:* <<cmd-" $1 "," $1 ">>", $2, "+"
+	sub("\"\"", "n/a", $3)
+	print "*Example:*", $4, "+"
+	print "****"
+	print "\n" $3, "+"
+	print "\n\n"
+}
```

Comment:
I'm not sure why, but the result includes the command name twice in the syntax line:

```
Syntax: save save <filename>
Example: save ~/important.txt
```

The last command, the one that jumps to a number, is broken altogether (too hard to describe with words, just look at the HTML).

Can this be amended somehow? If not, let's just roll back to the table that we had before.

Reply:
The save save and number thing shouldn't happen. If I had to guess this depends on the dsv file. I will check the one which is in my repository on GitHub. I did fix this already.

Reply:
Yes doc/cmdline-commands.dsv is wrong. Did I forgot a git add here maybe? I try again.

Reply:
Yup, that fixed it; thank you!